shell/common/shell_test.cc

```diff
@@ -402,5 +402,19 @@ bool ShellTest::IsAnimatorRunning(Shell* shell) {
   return running;
 }
 
+size_t ShellTest::GetLiveTrackedPathCount(
+    std::shared_ptr<VolatilePathTracker> tracker) {
+  size_t count = 0;
```

Comment:
Similar to above: this can be replaced by `std:count_if`

Reply:
Done